lua/lspconfig/fsautocomplete.lua

```diff
@@ -0,0 +1,61 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local bin_path = 'C:/tools/fsac/fsautocomplete.dll'
+if vim.fn.has('win32') == 0 then
+  bin_path = os.getenv('HOME')..'/lib/fsac/fsautocomplete.dll'
+end
+
+configs.fsautocomplete = {
+  default_config = {
+    cmd = {'dotnet', bin_path, '--background-service-enabled'};
+    root_dir = util.root_pattern('*.sln', '*.fsproj', '.git');
+    filetypes = {'fsharp'};
+    init_options = {
+      AutomaticWorkspaceInit = true;
+    };
+    settings = {};
+  };
+  docs = {
+    description = [[
+https://github.com/fsharp/FsAutoComplete
+
+Language Server for F# provded by FsAutoComplete (FSAC).
+
+Download a release of FsAutoComplete from [here](https://github.com/fsharp/FsAutoComplete/releases).
+It can also be built from sources. [Instructions](https://github.com/fsharp/FsAutoComplete) are on the main page.
+
+FsAutoComplete requires the [dotnet-sdk](https://dotnet.microsoft.com/download) to be installed.
+
+You may also need to configure the filetype as Vim defaults to Forth for `*.fs` files
+(unless you use something like vim-polyglot):
```

Comment:
probably obvious but maybe link to vim-polyglot

Reply:
reworded that whole bit to include all the plugins in a separate sentence.